File: DS_Study/Tree/Node.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Tree
{
    public class Node
    {
        public Node left;
        public Node right;
        public Node parent;
        public int value;

        public Node(int ivalue)
        {
            left = null;
            right = null;
            parent = null;
            value = ivalue;
        }

        public Node()
        {

        }

    }
}

File: Snapdeal/src/tree/Node.java
package tree;

public class Node {
	
	public Node leftNode = null;
	public Node rightNode = null;
	public int value;
	
	public Node() {
		
	}
	
	public Node(Node left, Node right, int val) {
		leftNode = left;
		rightNode = right;
		value = val;		
	}
	
	

}
